core/tasks.py

```diff
@@ -204,7 +204,12 @@ def queryIter(self, *args, **kwargs):
             logging.error("Could not continue queryIter - %s not known on this instance" % data["classID"])
         MetaQueryIter._classCache[data["classID"]]._qryStep(data)
 
-    queryIter.exposed = True
+    try:
+        queryIter.viur_flags
+    except AttributeError:
+        queryIter.viur_flags = {}
+    finally:
+        queryIter.viur_flags["exposed"] = True
```

Comment:
```suggestion
        decorators.ensure_viur_flags(queryIter)
        queryIter.viur_flags["exposed"] = True
```
Can we import `ensure_viur_flags` and simplyfy it ?

Reply:
@ciansen Please replace all old `.exposed = True` with the `@exposed` decorator --  That's why we have it.